h2/src/main/org/h2/engine/Constants.java

```diff
@@ -401,6 +401,16 @@
      */
     public static final int SALT_LEN = 8;
 
+    /**
+     * The identity of INFORMATION_SCHEMA.
+     */
+    public static final int META_SCHEMA_ID = -1;
```

Comment:
Can we call this INFORMATION_SCHEMA_ID, META_SCHEMA_ID makes me thinks of the special MetaTable we use.